packages/eslint-config/shared.js

```diff
@@ -151,5 +163,9 @@ module.exports = {
         jest: true,
       },
     },
+    {
+      files: ['web/src/**/*.routeHooks.{js,ts}'],
+      rules: { 'no-restricted-imports': 'off' },
```

Comment:
Maybe we should have our rwjs/internal rule here instead.

Reply:
Or..... just add `!web/src/**/*.routeHooks.{js,ts}` to the files pattern above

Reply:
I think you put your rules in the right place. Having your rules in `/.eslintrc.js` means that they will only apply to the framework, which seems to be what you'd want.

Actually, now I'm thinking maybe I should move my rules to `eslint-config/index.js` so they're only active in RW projects (not the framework).

Where did you want me to put `!web/src/**/*.routeHooks.{js,ts}`?
I don't think I can put it here
```
'no-restricted-imports': [
  'error',
  {
    patterns: [
      {
        group: ['$api/*'],
        message:
          'Importing from $api is only supported in *.routeHooks.{js,ts} files',
      },
    ],
  },
],
```
Because "group" is what the import string looks like in the file, it's not the path to the file